camino-messenger-bot-supplier.yaml.example

```diff
@@ -0,0 +1,46 @@
+# Example config for local development
+developer_mode: true
+
+matrix_key: PrivateKey-YOUR_PRIVATE_KEY
+matrix_host: matrix.camino.network
+matrix_store: mautrix.supplier.db
+
+rpc_server_port: 9090
+rpc_unencrypted: true
+rpc_server_cert_file: server-cert.pem                    #required if rpc_unencrypted: false
+rpc_server_key_file: server-key.pem                      #required if rpc_unencrypted: false
+rpc_url: https://columbus.camino.network/ext/bc/C/rpc    #required if supporting MintRequest
+
+partner_plugin_host: localhost
+partner_plugin_port: 50051
+partner_plugin_unencrypted: true
+partner_plugin_ca_file: ca-cert.pem     #required if partner_plugin_unencrypted: false
+
+response_timeout: 10000 # in milliseconds
+supported_request_types:
+  - ActivitySearchRequest
+  - ActivityProductListRequest
+  - AccommodationProductInfoRequest
+  - AccommodationProductListRequest
+  - AccommodationSearchRequest
+  - GetNetworkFeeRequest
+  - GetPartnerConfigurationRequest
+  - MintRequest
+  - ValidationRequest
+  - PingRequest
+  - TransportSearchRequest
+
+supplier_name: SUPPLIER_NAME #required only for Supplier bot if supporting MintRequest
+
+evm_private_key: PrivateKey-YOUR_PRIVATE_KEY # used for signing transactions on the c-chain / can be the same as the matrix_key
+evm_await_tx_confirmation_timeout: 6000 # timeout in milliseconds for awaiting transaction confirmation
```

Comment:
@evlekht I believe this variable is not used anymore. Before it was used here: https://github.com/chain4travel/camino-messenger-bot/blob/a22d8ee29bac0e4ddf576f0ba1c75fe3198bdb3a/internal/tvm/client.go#L73

Reply:
I see. So, this points into direction that this type of return error were explicitly part of t-client logic.
Regardless, since this field is now only used in error text, basically providing false info, I'd say we should remove both field and context deadline error - just treat it as other error types. We will look into it deeper when we'll do refactoring (really soon, I hope)
wdyt? regarding both removals

Reply:
I agree to both.

Reply:
I removed tx await timeout error and related stuff.

Reply:
@evlekht please resolve this conversation if you think the changes are OK.